File: docs/components_page/components/card/sizing/grid.R
library(dashBootstrapComponents)

first_card <- dbcCard(
  dbcCardBody(
    list(
      html$h5("Card title", className = "card-title"),
      p("This card has some text content, but not much else"),
      dbcButton("Go somewhere", color = "primary")
    )
  )
)

second_card <- dbcCard(
  dbcCardBody(
    list(
      html$h5("Card title", className = "card-title"),
      p(
        paste(
          "This card also has some text content and not much else, but",
          "it is twice as wide as the first card."
        )
      ),
      dbcButton("Go somewhere", color = "primary")
    )
  )
)

cards <- dbcRow(
  list(
    dbcCol(first_card, width = 4),
    dbcCol(second_card, width = 8)
  )
)

File: docs/components_page/components/card/sizing/grid.jl
using DashBootstrapComponents, DashHtmlComponents

first_card = dbc_card(
    dbc_cardbody([
        html_h5("Card title", className = "card-title"),
        html_p("This card has some text content, but not much else"),
        dbc_button("Go somewhere", color = "primary"),
    ]),
);


second_card = dbc_card(
    dbc_cardbody([
        html_h5("Card title", className = "card-title"),
        html_p(
            "This card also has some text content and not much else, but " *
            "it is twice as wide as the first card.",
        ),
        dbc_button("Go somewhere", color = "primary"),
    ]),
);


cards = dbc_row([dbc_col(first_card, width = 4), dbc_col(second_card, width = 8)]);
